src/common/sign-in/sign-in.tsx

```diff
@@ -23,6 +28,29 @@ export const SignIn = () => {
   const [emailState, setEmailState] = useState<EmailType>(initEmailState);
   const [passState, setPassState] = useState('');
   const [passHidden, setPassHidden] = useState(true);
+  const navigate = useNavigate();
+
+  const notifyOk = () => toast.success('Usuario loggeado correctamente');
+  const notifyErr = () => toast.error('Error');
+
+  const loginUser = async () => {
+    try {
+      const userSignIn: SignIn = {
+        email: emailState.inputValueEmail,
+        password: passState,
+      };
+
+      await SignInController.signIn(userSignIn);
```

Comment:
what do you think about add a spinner to notify the user that the api is called?